OrbitSsh/Channel.cpp

```diff
@@ -116,7 +116,7 @@ ResultType Channel::Read(std::string* result, int buffer_size) {
   return ResultType::kAgain;
 }
 
-ResultType Channel::Write(const std::string& text, int* written_length) {
+ResultType Channel::Write(const std::string& text, uint* written_length) {
```

Comment:
yeah, you probably want size_t here... and below

Reply:
true, thank you